pkg/api/vminfo.go

```diff
@@ -351,3 +362,19 @@ func (mem VmMemInfo) Format(state fmt.State, verb rune) {
 		state.Write([]byte(fmt.Sprintf("{Min:%d Max:%d Use:%d SlotSize:%v}", mem.Min, mem.Max, mem.Use, mem.SlotSize)))
 	}
 }
+
+func (cfg VmConfig) Format(state fmt.State, verb rune) {
+	// same-ish style as for VmInfo, differing slightly from default repr.
```

Comment:
How is it different? I see comment above talking about `resource.Quantity`. But if `VmConfig` doesn't have any `resource.Quantity`, maybe we can have the default formatting here?

Reply:
The main difference from the default representation is that for `%v` we're still including the field names. I don't know if it's honestly necessary though tbh